src/containers/IntentService.jsx

```diff
@@ -121,6 +122,20 @@ class IntentService extends Component {
     })
   }
 
+  alertSuccess(messages) {
+    const { t } = this.context
+
+    Notifier.info([
+      messages
+        .map(item => {
+          return t(item.message, item.params)
+        })
+        .join('.\n')
+    ])
+
+    this.setState({ konnector: null })
```

Comment:
I am not a big fan of having a `this.setState()` in an `alertSuccess` method. I think we should remove it as it is already called in `onSuccess`. Am I right ? What do you think ?

Reply:
Yep, totally, that's a mistake

Reply:
In fact alertSuccess is called only on `handleDeleteSuccess` so it should probably named alertDeleteSuccess.

And alert message doesn't work on the intent, so we can just make the setState without calling the Notifier